jetstream/leafnodes.md

```diff
@@ -0,0 +1,688 @@
+# Jetstream in LeafNodes 
+
+If you want to see a demonstration of the full range of this functionality, check out our [video](https://youtu.be/0MkS_S7lyHk)
+
+One of the use cases for leaf nodes is to provide a local nats network even when the connection to a hub or the cloud is down.
+To support such a disconnected use case with JetStream, independent JetStream islands are also supported and available through the same nats network.
+
+The general issue with multiple, independent JetStreams, accessible from the same client is that you need to be able to tell them apart. 
+As an example, consider a leaf node with a non clustered JetStream in each server. 
+You connect to one of them, but which JetStream responds when you use the JetStream API `$JS.API.>` ?
+
+To disambiguate that, the option `domain` was added to the JetStream configuration block. 
+When using it, follow these rules:
+Every server in a cluster and super cluster needs to have the same domain name. 
+This means that domain names can only change between two servers if they are connected via a leaf node connection.
+As a result of this the JetStream API `$JS.API.>` will also be available under a disambiguated name `$JS.<domain>.API.>`.
```

Comment:
Just a thought. Why don't we already have an export that is `$JS.<account>.API.>` instead of making another unique token....

Reply:
because an account can run in multiple domains.